Packs/CrowdStrikeFalconX/Integrations/CrowdStrikeFalconX/CrowdStrikeFalconX.py

```diff
@@ -0,0 +1,685 @@
+from typing import Dict, Callable, Tuple, Any, Union, List
+
+from requests import Response
+
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+
+import requests
+import traceback
+import shutil
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+
+def convert_environment_id_string_to_int(environment_id: str) -> int:
+    """
+    Converting the string that describes the environment id into an int which needed for the http request
+    :param environment_id: one of the environment_id options
+    :return: environment_id represented by an int
+    """
+    environment_id_options = {
+        "300: Linux Ubuntu 16.04": 300,
+        "200: Android (static analysis)": 200,
+        "160: Windows 10": 160,
+        "110: Windows 7": 110,
+        "100: Windows 7": 100,
+        "64-bit": 64,
+        "32-bit": 32,
+    }
+    return environment_id_options.get(environment_id)
+
+
+class Client(BaseClient):
+    """
+    Client to use in the CrowdStrikeFalconX integration. Uses BaseClient
+    """
+
+    def __init__(self, server_url: str, username: str, password: str, use_ssl: bool, proxy: bool):
+        super().__init__(base_url=server_url, verify=use_ssl, proxy=proxy)
+        self._username = username
+        self._password = password
+        self.token = self._generate_token()
```

Comment:
if `token` used only in the class, convention in PEP8 is to prefix with underscore private traits.
```suggestion
        self._token = self._generate_token()
```